DemoServer/AdditionalLanguages/java/src/main/java/net/ravendb/demo/textSearch/fTSQuerySearchWildcards/metadata.json

```diff
@@ -0,0 +1,188 @@
+        },
+        {
+            "Title": "Full text searching - Book",
+            "Type": "link",
+            "Url": "https://ravendb.net/learn/inside-ravendb-book/reader/4.0/9-querying-in-ravendb#full-text-searching"
+        },
+        {
+            "Title": "How to search",
+            "Type": "link",
+            "Url": "https://ravendb.net/docs/article-page/latest/java/client-api/session/querying/how-to-use-search"
+        },
+        {
+            "Title": "Querying using wildcards",
+            "Type": "link",
+            "Url": "https://ravendb.net/docs/article-page/latest/java/indexes/querying/searching#using-wildcards"
+        }
+    ],
+
+    "Walkthroughs": [
+        {
+            "Title": "Define a query",
+            "Slug": "step-1",
+            "DescriptionLines": [
+                "* Define a query on the 'LastFms' collection using the Session `query()`."
+            ],
+            "Assets": [
+                {
+                    "Title": "Query Overview Demo",
+                    "Type": "demo",
+                    "Url": "/java/queries/query-overview"
+                }
+            ]
+        },
+        {
+            "Title": "Define search criteria I",
+            "Slug": "step-2",
+            "DescriptionLines": [
+                "* Use `search()` to search for the specified terms in the specified document field.",
+                "",
+                "* In this example,<br>",
+                "  we search for documents that have a term with _'start' param prefix_ AND _'end' param postfix_,",
+                "  within their _Artist_ field.",
+                "",
+                "* Note - as opposed to searching with _wildcard as a postfix_ - <br>",
+                "  searching with _wildcard as the prefix_ of the term (i.e. '*text') is Not recommended,<br>",
+                "  as it will cause RavenDB to perform a full scan instead of an efficient prefix search."
+            ],
+            "Assets": [
+                {
+                    "Title": "FTS Query - Basics Demo",
+                    "Type": "demo",
+                    "Url": "/java/text-search/fts-query-search-basics"
+                },
+                {
+                    "Title": "FTS Query - Search Operations Demo",
+                    "Type": "demo",
+                    "Url": "/java/text-search/fts-query-search-operators"
+                },
+                {
+                    "Title": "How to search",
+                    "Type": "link",
+                    "Url": "https://ravendb.net/docs/article-page/latest/java/client-api/session/querying/how-to-use-search"
+                },
+                {
+                    "Title": "Querying searching",
+                    "Type": "link",
+                    "Url": "https://ravendb.net/docs/article-page/latest/java/indexes/querying/searching"
+                }
```

Comment:
Are the above 2 links working ? 
or do you want to add 'TODO' here for them, as was done in walkthrough step number 3 ?

Reply:
@Drorlev 
For the future:
Pls write `'done'` for any comment when you apply the fix.